### DISCUSSION TITLE
Symfony 5 support ?

### DISCUSSION THREAD

**Discussion by bastien70** [deployphp/deployer#2313] [Help needed]
Hello, when will the release taking into account Symfony 5 take place?

Currently there are things that come from symfony 3 like:

`set('shared_files', ['app/config/parameters.yml']);` => now, we have .env

`assets` => currently, the `deploy:assets:install` command is `run('{{bin/php}} {{bin/console}} assets:install {{console_options}} {{release_path}}/web'); `. Now, it must be `run('{{bin/php}} {{bin/console}} assets:install {{console_options}} {{release_path}}/public'); `

And there are surely other little things that have changed today with Symfony 5.

I understood that a BETA was in progress? Does it take these changes into account? When will we have an official and stable version taking into account the changes for Symfony 5?

The file to change : https://github.com/deployphp/deployer/blob/6708d2640f9d8b68b539b3929d9b09cf0d4fec1e/recipe/symfony.php#L95

**Comment by antonmedv** [comment]
Please take a look at master branch. There recipe what will be in soon v7 release.
